src/modules/recipe/lightDomStylesChild/lightDomStylesChild.css

```diff
@@ -0,0 +1,4 @@
+/* These styles apply to this Light DOM component, and also to the parent component */
+.unscoped {
```

Comment:
Is there a way that we can mark this as "danger zone"? Generally speaking light DOM uses mostly scoped styles, so this is more an anti-pattern.

Reply:
The documentation says Light DOM doesnt scope the styles, and if we need scoping we must use scoped styles - https://lwc.dev/guide/light_dom#css.

I can add a line that says "As a best practice, we recommend using scoped styles where possible".

Reply:
Yup, a byline like that is what I was thinking about. Generally speaking I believe that most won't have unscoped styles.